map_frame: world

# pixel map
pixel_map:
  # min coordinates in world frame [m]
  xy_min: [-100.0, -100.0]  # [m]

  # max coordinates in world frame
  xy_max: [100.0, 100,0]  # [m]

  # resolution
  resolution: 0.5  # [m]

  # initial value of cells
  init_value: 0.0

  # cell value bounds
  bound_min: 0.0
  bound_max: 1.0

  # absolute position pointcloud cropping
  min_z_abs: 0.5  # [m]
  max_z_abs: 3.0  # [m]

  # relative position pointcloud cropping
  use_rel_flatten: false
  min_z_rel: -0.5  # [m]
  max_z_rel: 0.5  # [m]

  # publishing
  publish_map: false

# voxel map
voxel_map:
  # min coordinates in world frame
  xyz_min: [-200.0, -200.0, -5.0]  # [m]

  # max coordinates in world frame
  xyz_max: [200.0, 200.0, 30.0]  # [m]

  # resolution
  resolution: [0.5, 0.5, 0.5]  # [m]

  # initial value of cells
  init_value: 0.0

  # cell value bounds
  bound_min: 0.0  # [m]
  bound_max: 1.0  # [m]

  # absolute position pointcloud cropping
  min_z_abs: -5.0  # [m]
  max_z_abs: 30.0  # [m]

  # relative position pointcloud cropping
  use_rel_cropping: false
  min_z_rel: -0.5  # [m]
  max_z_rel: 0.5  # [m]

  # probabilistic increments
  hit_inc: 0.2
  miss_inc: -0.01

  # publishing
  publish_map: true
